specification/implementation/metadata/mets/amdsec/index.md

```diff
@@ -1,9 +1,16 @@
 ### 5.3.4.	Use of the METS administrative metadata section (element amdSec)
-The purpose of the METS administrative data section is to embed or refer to files containing administrative metadata about the IP content objects. CSIP is only using referencing of files containing administrative metadata. The CSIP (and METS) categorises preservation metadata as administrative metadata, specifically Digital Provenance metadata (following the avaiable guidelines), hence all preservation metadata should be referenced from a digiprovMD element within the amdSec.
+The purpose of the METS administrative metadata section is to embed or refer to files containing administrative metadata about the IP content objects. CSIP is only using referencing of files containing administrative metadata. 
 
-The administrative metadata section also contains rights statements. Regardless of used standard used to record the rights metadata it should be referenced from a rightsMD element within the amdSec. Suitable standards include the collaborative work from RIGHTSTATMENTS.ORG <http://rightsstatements.org> , Europena right statements <https://pro.europeana.eu/page/available-rights-statements> , the METS Rights Schema <https://github.com/mets/METS-Rights-Schema> created and maintained by the METS Board, the rights sections of PREMIS <http://www.loc.gov/standards/premis/> as well as any local rights statements.
+The administrative metadata section contains four sections technical metadata (element techMD)>, rights metadata (element rightsMD), source metadata (element sourceMD) and digital provenance metadata (element digiprovMD). The techMD element records technical metadata about content files, rightsMD records intellectual property rights information, sourceMD records descriptive, technical or rights information about an analog source document used to generate the digital library object, and digiprovMD records digital preservation information, such as information about the digital library object's life-cycle and history.
```

Comment:
The administrative metadata section contains four sub-sections each used to record different types of metadata for package content:
 - technical metadata (element techMD) records technical metadata;
 - rights metadata (element rightsMD) records intellectual property rights information;
 - source metadata (element sourceMD) records descriptive, technical or rights metadata for an analog source for a digital library object; and
 - digital provenance metadata (element digiprovMD) records digital preservation information, e.g. audit information covering a digital library object's life-cycle.